fetch_description/robots/fetch.urdf

```diff
@@ -587,7 +587,7 @@
       </geometry>
     </collision>
   </link>
-  <joint name="torso_fixed_joint" type="fixed">
+  <joint name="torsoed_joint" type="fixed">
```

Comment:
@RDaneelOlivav sorry I didn't specify before, but there were multiple instances needing to be reverted to the original.

Reply:
I thought I had fixed that in this commit: 47b97450cb26776d44131fb847ca52a42d093c29

Is there something I'm missing here? There were two places where there was an error. Is there anywhere else?

Reply:
Yeah, this is weird. Your commit's (the most recent one) diffs look correct, but when viewing the full diffs, the current version of the file is wrong for this line. I can easily fix that and am doing so now.

Reply:
There were actually three places, I see now. :+1: